src/main/java/org/folio/rest/impl/InitAPIImpl.java

```diff
@@ -24,13 +31,34 @@ public void init(Vertx vertx, Context context, Handler<AsyncResult<Boolean>> han
       result -> {
         if (result.succeeded()) {
           registerProxies(context);
```

Comment:
please call registerProxies(context) and initializeSystemProperties(handler) right after SpringContextUtil.init.  
vertx.executeBlocking method catches any exceptions in the first lambda